mainwindow.cpp

```diff
@@ -212,3 +213,25 @@ void MainWindow::on_saveImage_clicked()
     /// Save the png image at the location (path) selected previously.
     img.save(path);
 }
+
+
+void MainWindow::on_limitsButton_clicked()
+{
+    QMessageBox::information(this, "Limits", "Definition: \n\nLet f be a function defined on some open interval that contains the number a, except possibly at a itself. We say the limit of f(x) as x approaches a is L, and we write lim f(x) as x -> a = L if for every number epsilon > 0 there is a corresponding number theta > 0 such that |f(x)-L| < epsilon whenever 0 < |x-a| < delta \n\nLeft-Hand [or Right-Hand] Limit: \n\nLet f be a function defined on some open interval (b,a) [or (a,b)]. We say the left-hand [or right-hand] limit of f(x) as x approaches a is L, (or the limit of f(x) as x approaches a from the left [or right] is L) and we write lim x->a- f(x) = L[lim x->a+ f(x) = L] if for every number epsilon > 0 there is a corresponding number delta > 0 such that |f(x)-L| < epsilon whenever a-epsilon < x < a[a < x < a + epsilon] ");
```

Comment:
Indentation. Move sentences on new lines to remove the long horizontal scroll bar.